Fastfile

```diff
@@ -150,12 +150,11 @@ def _build_with_gym
   export_method = _get_export_method
   xcconfig_filename = Dir.pwd + '/TAB.release.xcconfig'
   create_xcconfig(filename: xcconfig_filename)
-  gym(configuration: _configuration, export_method: export_method, xcconfig: xcconfig_filename)
+  gym(configuration: _get_build_config, export_method: export_method, xcconfig: xcconfig_filename)
 end
 
-def _configuration
-  value = get_info_plist_value(path: ENV['GYM_EXPORT_OPTIONS'], key: 'configuration')
-  return value.nil? ? "Release" : value
+def _get_build_config
+  return ENV['BUILD_CONFIGURATION'].nil? ? "Release" : ENV['BUILD_CONFIGURATION']
```

Comment:
To align with existing naming convention, I wonder if `FL_BUILD_CONFIGURATION` might be better?

Reply:
We are not making use of Fastlane to update the value, not sure we want to specify FL having a role in the env var.

Reply:
I guess fastlane is consuming the value so it can have FL as a prefix, your call mate.

Reply:
Changed to FL_BUILD_CONFIGURATION